develop/smart-contracts/block-explorers/.pages

```diff
@@ -1,4 +1,3 @@
 title: Block Explorers
 nav:
-  - index.md
-  - 'Contract Verification': contract-verification.md
\ No newline at end of file
+  - index.md
```

Comment:
If there's no content in this section, we should just get rid of it

Reply:
We left it like that because we need to add the contract verification pages. We hid them because they are not currently working in Passet Hub.